trie/verkle.go

```diff
@@ -258,12 +258,22 @@ func (trie *VerkleTrie) Commit(_ bool) (common.Hash, *NodeSet, error) {
 	}
 
 	batch := trie.db.diskdb.NewBatch()
+	const keyPrefix = "flat-"
+	path := make([]byte, 0, len(keyPrefix)+32)
```

Comment:
`32` is probably more than needed, but is a bit irrelevant. Leaning on 100% correctness.
Probably `10` should be 99.999% fine. Also, I'm using `append` instead of `copy` so in the worst case would be 100% correct, but anyway.